app/marketing/management/commands/assemble_leaderboards.py

```diff
@@ -214,6 +216,10 @@ def handle(self, *args, **options):
         for key, rankings in ranks.items():
             rank = 1
             for username, amount in sorted(rankings.items(), key=lambda x: x[1], reverse=True):
+                try:
+                    tech_keywords = profile_keywords_helper(username)
```

Comment:
seems fairly costly to pull these down every night.  im pretty sure we'll get rate limited by github if we do it this way

can we the keywords in a list on the `Profile` model, and only make a remote call IFF they are empty on that model?

note to self: we should sync the keywords of Profiles that aren't just on the leaderboard.  should add a call to this method in `sync_profiles.py`

Reply:
ook, it'll save a LOT of requests